### ISSUE TITLE
[v1.x] [v2.0] Line doesn't copy if instruction dragged and dropped above it

### ISSUE THREAD

**Issue by danil-kondr2016** [kodemeister/X584#39]
https://github.com/user-attachments/assets/5790e7e7-662c-4eb5-8fb9-99e9e95d3e9d

**Comment by danil-kondr2016** [comment]
It seems that CodeListView is inactive when Copy button is pressed (and also Cut, Paste, Delete).

**Comment by kodemeister** [comment]
Yes. When you drag microinstructions from `CodeTreeView` to `CodeListView`, it seems like `CodeTreeView` still remains active. That's why Copy, Cut, Paste, Delete don't work.

To verify this, first revert all hacks introduced in 448b68134e817eab7731f444f229e717e4ded8c0:

```cpp
if ((ActiveControl == CodeListView || ActiveControl == CodeTreeView) && !InputEdit->Visible) {
```

to the original code:

```cpp
if (ActiveControl == CodeListView && !InputEdit->Visible) {
```

Then manually change keyboard focus to `CodeListView` in `CodeTreeViewDblClick`:

```cpp
CodeListView->SetFocus();
```

If this works both for 1.x and 2.0, please open a PR against 1.x.

**Comment by kodemeister** [comment]
Fixed by #41 (1.x for now).
